src/04-arrays.js

```diff
@@ -24,7 +27,9 @@ function createEmptyArray() {}
  *  createArrayWithTwoElements(true, false);
  *  //> [ true, false ]
  */
-function createArrayWithTwoElements(a, b) {}
+function createArrayWithTwoElements(a, b) {
+  return [a, b];
```

Comment:
Great job returning the array literal, rather than going through extra steps! Very compact solution.